.travis.yml

```diff
@@ -6,3 +6,7 @@ rust:
 matrix:
   allow_failures:
     - rust: nightly
+script:
+  - cargo build --no-default-features
```

Comment:
How about also enabling `--verbose` here?

Reply:
Do we need to test this at all?

Reply:
I agree with the `--verbose` flag. And I would say yes. There might be some forgotten `#[cfg(feature="<<flag>>"]` missing that makes it not compile